prefixer/composer.json

```diff
@@ -30,7 +30,8 @@
       "php fix-twig.php",
       "php fix-doctrine.php",
       "php fix-swiftmailer.php",
-      "php fix-validator.php"
+      "php fix-validator.php",
```

Comment:
We could start to think of a more sophisticated "cleaner" than those copy-pasted scripts 🤔

Reply:
Hmmm, maybe we could. There is a copy-pasted error handling code snippet. On the other hand, what we have is a simple solution and one can see easily from composer.json that there are packages and some scripts which modify them.